collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java

```diff
@@ -0,0 +1,19 @@
+package org.odk.collect.android.utilities;
+
+import android.widget.Toast;
+
+import org.odk.collect.android.application.Collect;
+
+/**
+ * Created by shobhit on 6/3/17.
+ */
+
+public class ToastUtils {
+    public static void shortDuration(String message){
+        Toast.makeText(Collect.getInstance(),message, Toast.LENGTH_SHORT).show();
```

Comment:
Space after comma. A better name might be makeShortText. Perhaps there should a third, private method that factors out the common code.